iOS_SDK/OneSignalSDK/Source/OneSignalHelper.m

```diff
@@ -358,8 +379,10 @@ + (void)handleNotificationAction:(OSNotificationActionType)actionType actionID:(
     
     [OneSignalTrackFirebaseAnalytics trackOpenEvent:result];
     
-    if (!notificationOpenedHandler)
+    if (!notificationOpenedHandler) {
+        [[self getUnprocessedOpenedNotifis] addObject:result];
```

Comment:
maybe have this addObject in a method, to have the UnprocessedOpenedNotifis mutations centralized?

Reply:
Created fixup commit 4e0fc0c2b46a1660a4d3b706c161354e85f682f2 to address this.